### DISCUSSION TITLE
cant use Lightweight snapshot

### DISCUSSION THREAD

**Discussion by dachengcheng2022** [filecoin-project/lotus#7862] [Build and Sync] [answered]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
lotus version 1.13.2-rc4+mainnet+git.e05fdf892.dirty
```

### Describe the Bug

ERROR: chain validation failed: getting block messages for tipset: failed to get messages for block: failed to load msgmeta (bafy2bzacecmwp4imjqhdg2zvc7j2s4xxahnn5jnudtrt335re24i4zim7ccfi): blockstore: block not found

### Logging Information

```text
ERROR: chain validation failed: getting block messages for tipset: failed to get messages for block: failed to load msgmeta (bafy2bzacecmwp4imjqhdg2zvc7j2s4xxahnn5jnudtrt335re24i4zim7ccfi): blockstore: block not found
```

### Repo Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

**Comment by Reiers** [comment]
Hi @lichegn 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to [Lotus Discussions](https://github.com/filecoin-project/lotus/discussions) for help and troubleshooting.

Closing Lotus Issue ticket.

**Comment by Reiers** [comment, marked as answer]
I noticed that you running on .dirty version.
`lotus version 1.13.2-rc4+mainnet+git.e05fdf892.dirty`

I will suggest that build Lotus again:
https://github.com/filecoin-project/lotus/releases/tag/v1.13.2-rc4

Build with CUDA:
```
cd lotus/
git pull
git checkout v1.13.2-rc4
git submodule update
RUSTFLAGS="-C target-cpu=native -g" FFI_BUILD_FROM_SOURCE=1 FFI_USE_CUDA=1 make clean all
sudo make install
```

Build with OpenCL:
```
cd lotus/
git pull
git checkout v1.13.2-rc4
git submodule update
RUSTFLAGS="-C target-cpu=native -g" FFI_BUILD_FROM_SOURCE=1 make clean all
sudo make install
```

**Solution:**

1. Delete chain folder in your .lotus path
2. Download snaphot like this: `wget https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car`
3. `lotus daemon --import-snapshot <filefromstep1> --halt-after-import`
4. Wait until import is complete, _you will know when process has ended._
5. Start up again with `lotus daemon`

This solution was without verification, and it should be fine!
But if you want to do it with verification follow the same steps from our Docs:
https://lotus.filecoin.io/docs/set-up/chain-management/#restoring-a-custom-snapshot
just with verification. 

I reproduced the issue myself and it worked fine here.
**If the issue still persist - comment, and we will continue to troubleshoot.**
